Templates/blog/modifyUser.html.twig

```diff
@@ -0,0 +1,90 @@
+{% extends "base.html.twig" %}
+{% block body %}
+<body id="page-top">
+<!-- Post creation Section-->
+<section class="page-section" id="contact">
+    <div class="container">
+        <!-- Contact Section Heading-->
+        <h2 class="page-section-heading text-center text-uppercase text-secondary mb-0">Modifier le user</h2>
+        <!-- Icon Divider-->
+        <div class="divider-custom">
+            <div class="divider-custom-line"></div>
+        </div>
+        <!-- Contact Section Form-->
+        <div class="row justify-content-center">
+            <div class="col-lg-8 col-xl-7">
+                {% if user %}
+                <!-- * * * * * * * * * * * * * * *-->
+                <form action="/user/update/{{ user.id }}" method="post">
+                    <!-- Firstname input-->
+                    <div class="form-floating mb-3">
+                        <input class="form-control" type="text" id="firstname" name="firstname" placeholder="Prénom" value="{{ user.firstname }}" required />
+                        <label for="firstname">Prénom</label>
+                    </div>
+                    {% if checkErrors.firstname %}
+                        {% for key, value in checkErrors.firstname %}
+                            <div class="alert alert-danger" role="alert">
+                                Firstname : {{ value }} <br>
+                            </div>
+                        {% endfor %}
+                    {% endif %}
```

Comment:
instead of using alert , you could use "is-invalid" class on invalid fields and use "invalid-feedback" in div to display errors like we can see in bootstrap 5 exemples.  https://getbootstrap.com/docs/5.0/forms/validation/#server-side